ansible/inventory/vagrant

```diff
@@ -9,5 +9,8 @@ node-1
 node-2
 node-3
 
+[swarm_master]
+node-4
```

Comment:
Please make `node-1` as `swarm_master`. This is not a mistake, but we will try to keep master services from nodes starting from 1.